exchange/targeting_test.go

```diff
@@ -57,9 +59,25 @@ func assertKeyExists(t *testing.T, bid *openrtb.Bid, key string, expected bool)
 	}
 }
 
+// Verify we do not set winning bid keys if includewinners is set false
```

Comment:
Can we move this into an `exchange/exchangetest`? You can basically copy/paste/tweak the `targeting-with-winners.json` one.

I realized while working on some recent features that all these tests of private methods make the `exchange` code really hard to maintain. Since the `exchangetest` files are independent of `exchange` implementation, we can move around the private code without having to update the tests... which in turn makes refactors much safer.